task-header.html

```diff
@@ -0,0 +1,34 @@
+<html>
+	<head>
+		<link rel="stylesheet" href="task-header.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	</head>
+	
+	<body>
+		<nav class="firstnav">
+			<div class="infobar">
+				<a href="#"> About Me </a>
```

Comment:
Might be worth adding names to the # just for practice.
```suggestion
				<a href="#aboutme"> About Me </a>
```

Reply:
I was going to leave them # until I played around with adding links. But see the value in stating what the href's are.